src/validation.cpp

```diff
@@ -1457,7 +1457,7 @@ PackageMempoolAcceptResult ProcessNewPackage(CChainState& active_chainstate, CTx
     assert(std::all_of(package.cbegin(), package.cend(), [](const auto& tx){return tx != nullptr;}));
 
     std::vector<COutPoint> coins_to_uncache;
-    const CChainParams& chainparams = Params();
+    const CChainParams& chainparams = active_chainstate.m_params;
```

Comment:
5c67e84d37d452e9186a6357e5405fabeff241c7 nit: `.m_chainman.GetParams()`

Seems odd to add a TODO and violate it in the same commit (     /* TODO: replace with m_chainman.GetParams() */)